src/__tests__/index.test.ts

```diff
@@ -43,7 +43,24 @@ const testNotificationUnseen: InappNotification = {
   title: 'You have a new comment',
   date: new Date().toISOString()
 };
-
+const closePastDate = Date.now() - 10 * 1000;
```

Comment:
I just added your test cases to my code (without any code changes) and it passed. 

Can you explain why you are not following TDD?